src/utils/dispute-utils.js

```diff
@@ -372,7 +374,10 @@ function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
     },
     {
       phase: DisputesTypes.Phase.ConfirmAppeal,
-      endTime: confirmAppealEndTime,
+      endTime:
+        roundAppealed && appeal.opposedRuling > 0
+          ? appeal.confirmedAt
+          : confirmAppealEndTime,
       active:
```

Comment:
I get what you mean, but i think that they require different calculations because we can't rely on the `roundAppealed` variable to know if the phase is the active one. If the round was not appealed, it doesn't necessarily mean that is still active, because it could already be a past phase from a previous round.